packages/legend-graph/src/graphManager/Testable_PureGraphManagerPlugin_Extension.ts

```diff
@@ -51,4 +57,9 @@ export interface Testable_PureGraphManagerPlugin_Extension
    * Get the list of atomic test observers.
    */
   getExtraAtomicTestObservers?(): AtomicTestObserver[];
+
+  /**
+   * Get the list of atomic test observers.
+   */
```

Comment:
Can you please change the comment here according to what you added

Reply:
Sure, changed!